apps/docs/src/pages/components/panel.mdx

```diff
@@ -0,0 +1,54 @@
+import { HvPanel, panelClasses } from "@hitachivantara/uikit-react-core";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("Panel", "core", panelClasses);
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+### Usage
+
+```tsx live
+<div className="w-full h-100px">
+  <HvPanel className="flex w-full h-full items-center justify-center">
+    Panel Content
+  </HvPanel>
+</div>
```

Comment:
centering the text takes quit a bit of styles, and will almost never be what the user wants. what do you think of just having this?:

```suggestion
<div className="w-full">
  <HvPanel className="h-100px">
    Panel Content
  </HvPanel>
</div>
```

Reply:
yeah, seems reasonable. I'll update.